client/my-sites/promote-post-i2/components/posts-table/index.tsx

```diff
@@ -14,17 +15,33 @@ interface Props {
 export default function PostsTable( props: Props ) {
 	const { posts, isLoading, isFetchingPageResults } = props;
 
+	const testExperimentName = 'dsp_blaze_open_widget_button_202308';
```

Comment:
We probably don't need to use `testExperimentName` constant, so we can put the experiment name right to `useExperiment` call.